app/grants/management/commands/sync_pending_contributions.py

```diff
@@ -54,3 +56,65 @@ def handle(self, *args, **options):
 
             for contribution in zcash_pending_contributions.all():
                 sync_payout(contribution)
+
+
+        # PENDING CELO TXN
+        celo_pending_contributions = pending_contribution.filter(subscription__tenant='CELO')
+        if celo_pending_contributions:
```

Comment:
I believe `if Queryset` always returns true since it hasn't been evaluated. Normally you would use `.exists()` to see if a queryset returns any records without having to evaluate the whole thing. But in this case, I don't even think the `if` statements here are needed. Within the if statement you call update on a queryset and then iterate over it, both of which will just be no-ops if the queryset is empty.